code/game/turfs/simulated/walls_misc.dm

```diff
@@ -64,10 +64,12 @@
 /turf/closed/wall/clockwork/process()
 	if(prob(2))
 		playsound(src, 'sound/magic/clockwork/fellowship_armory.ogg', rand(1, 5), 1, -4, 1, 1)
-	for(var/obj/structure/clockwork/cache/C in range(1, src))
-		if(prob(5))
-			clockwork_component_cache[pick("belligerent_eye", "vanguard_cogwheel", "guvax_capacitor", "replicant_alloy", "hierophant_ansible")]++
+	for(var/obj/structure/clockwork/cache/C in orange(1, src))
```

Comment:
Eh, I can make the cache do the range check I guess but I want the walls to clang